README.md

```diff
@@ -8,6 +8,25 @@
 ElasticDL is a Kubernetes-native deep learning framework
 that supports fault-tolerance and elastic scheduling.
 
+**IMPORTANT**: This repository is deprecated.
+
+- The elasticity and fault-tolerance are no longer being actively maintained
```

Comment:
ElasticDL repository is no longer maintained. Users are encouraged to switch to [DLRover](https://github.com/intelligent-machine-learning/dlrover). 

In addition to the elasticity and fault-tolerance, DLRover also ....